src/Native/Bootstrap/CppCodeGen.h

```diff
@@ -41,4 +41,33 @@ inline double __uint64_to_double(uint64_t v)
     return val.d;
 }
```

Comment:
You can move ReversePInvokeFrame from common.h here as well while you are on it.

Reply:
(Most of the CppCodeGen specific stuff should really be in this file actually and not in common.h.)